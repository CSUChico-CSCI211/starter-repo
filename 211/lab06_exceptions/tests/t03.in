67
-4
6
