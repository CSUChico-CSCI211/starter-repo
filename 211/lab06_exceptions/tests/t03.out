Pick a number between 1 and 10.
You entered an illegal value of 67. Please try again.
Pick a number between 1 and 10.
You entered an illegal value of -4. Please try again.
Pick a number between 1 and 10.
You picked 6.
